index.html

```diff
@@ -134,8 +139,18 @@
           $("#jsonInput").val(JSON.stringify(testDataClinical, null, '    '));
         }
         $("#parseJson").on("click", function() {
-          //initialise with advanced view
-          initViews(true);
+           if (viewType === "simple") {
+            initViews(false);
+            if (location.href.indexOf("?view=") === -1) {  
+              window.history.pushState('simpleView', '', location.href+="?view=simple");
```

Comment:
Should the history.pushState make the forward/backword buttons on the browser work when switching between views? It doesn't atm